htdocs/cron/list.php

```diff
@@ -544,7 +544,7 @@
 		$object->datelastrun = $db->jdate($obj->datelastrun);
 		$object->datenextrun = $db->jdate($obj->datenextrun);
 
-		$datelastrun = $db->jdate($obj->datelastrun);
+		$datelastrun = (int)$db->jdate($obj->datelastrun);
```

Comment:
If $obj->datelastrun is empty or null, if you cast into int, $datelastrun will be 0 meaning 1970-01-01 when we want '' (meaning undefined).
So old code this good.